src/sdl2.nim

```diff
@@ -1390,6 +1390,23 @@ proc createCursor*(data, mask: ptr uint8;
 #*
 proc createColorCursor*(surface: SurfacePtr; hot_x, hot_y: cint): CursorPtr {.
   importc: "SDL_CreateColorCursor".}
+#*
+type SystemCursor* = enum

```

Comment:
Please stick to [NEP-1](https://github.com/nim-lang/Nim/wiki/Style-Guide-for-Nim-Code). ``type`` should be on its own line.